src/identity.ts

```diff
@@ -68,5 +78,12 @@ export default class Identity {
 
     this.network = (networks as Record<string, Network>)[args.chain];
     this.type = args.type;
+
+    // set the restorer if the user specified it.
+    if (args.restorer) {
+      this.restorer = args.restorer;
+    } else {
+      this.restorer = Identity.DEFAULT_RESTORER;
+    }
```

Comment:
```suggestion
    this.restorer = Identity.DEFAULT_RESTORER;
    if (args.restorer) {
      this.restorer = args.restorer;
    }
```



Else should be avoided in general if not necessary. https://www.yegor256.com/2016/08/10/if-then-else-code-smell.html

Reply:
very good article thnaks :)